gnovm/cmd/gno/testdata/lint/render_multiple_cardinality_errors.txtar

```diff
@@ -0,0 +1,22 @@
+# testing gno tool lint command: Render function with multiple cardinality errors (too many parameters and returns)
```

Comment:
I wanted to either have all these tests in a single file or have a `testdata/lint/render` but figured this is fine for now and can be done in a separate PR if thats needed to keep this one focused

Reply:
I'd say to have these as unit tests in `tool_lint_test.go`; they're definitely too "table-like" to have as integration tests.